packages/lb-annotation/src/utils/tool/CommonToolUtils.ts

```diff
@@ -109,32 +101,6 @@ export default class CommonToolUtils {
     return flag;
   }
 
-  /**
-   * 筛选当前的步骤配置
-   * @param toolName
-   */
-  public static getCurrentOperation(toolName: EToolName | ECheckModel) {
```

Comment:
Can you still retain the function `getCurrentOperation` in  `CommonToolUtils` ?  Some users is using `ToolUtils.getCurrentOperation` to get `toolOperation`